google/cloud/spanner_v1/types/data_types.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Custom data types for spanner."""
+
+
+class JsonObject(dict):
+    """
+    JsonObject type help format Django jsonfield to compatible Cloud Spanner's
```

Comment:
`jsonfield` - looks like a typo

Reply:
jsonfield is a datatype in django, corrected the case for the same.